seed/building_sync/mappings.py

```diff
@@ -0,0 +1,733 @@
+import copy
+from datetime import datetime
+
+import pytz
+from lxml import etree
+
+
+BUILDINGSYNC_URI = 'http://buildingsync.net/schemas/bedes-auc/2019'
```

Comment:
I'm guessing that this URI is going to be updated to 2020 when we release 2.1/3.0 this year (June). Any issues with that?

Reply:
That will probably a bit of refactoring, but I don;'t think it should be too bad. I will hold off on making those changes for now